components/03-modules/filter/_filter.scss

```diff
@@ -0,0 +1,144 @@
+$filter__transition                        : $transition-base !default;
+// Swatch
+$filter__swatch-margin                     : 0 0 $spacer--medium 0 !default;
+$filter__swatch-option-size                : 48px !default;
+$filter__swatch-option-margin              : $spacer--medium !default;
+$filter__swatch-option-background          : $white !default;
+$filter__swatch-option-color               : $gray-dark !default;
+$filter__swatch-option-border              : 3px solid $white !default;
+$filter__swatch-option-border-color-hover  : $color-primary !default;
+$filter__swatch-option-border-color--active: $color-primary !default;
+// Label
+$filter__label-padding                     : $spacer--medium 0 !default;
+// Color
+$filter__color-margin                      : 0 0 $spacer--medium 0 !default;
+$filter__color-option-width                : 56px !default;
+$filter__color-option-height               : 80px !default;
+$filter__color-option-after-top            : 4px !default;
+$filter__color-option-after-left           : 8px !default;
```

Comment:
shouldn't we use here `$spacer` ?

Reply:
Nope, this is positioning, no need to use spacer for this purpose.